### PR TITLE
Binary change

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: TA-sysmon-deploy-for-cyences/README.md [hunk 1]
```diff
@@ -64,11 +67,12 @@ CONTRIBUTORS
...
+* Mahir Chavda
 
 
 SUPPORT
 -------
 * Contact - CrossRealms International Inc.
   * US: +1-312-2784445
 * License Agreement - https://d38o4gzaohghws.cloudfront.net/static/misc/eula.html
-* Copyright - Copyright CrossRealms Internationals, 2021
+* Copyright - Copyright CrossRealms Internationals, 2023
```
Comment (VatsalJagani) [inline TA-sysmon-deploy-for-cyences/README.md:78]: great!!

File: TA-sysmon-deploy-for-cyences/default/inputs.conf [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+interval = 14400
 
 [script://.\\bin\\update.bat]
 disabled = False
-interval = 43200
+interval = 10800
+
+[script://.\\bin\\remove_log_files.bat]
+disabled = False
+interval = * * * * 0
```
Comment (VatsalJagani) [inline TA-sysmon-deploy-for-cyences/default/inputs.conf:12]: Can you keep it for every 2 hour, and other two inputs as every 10 minutes and keep it running for 24 hours and see how the logs are behaving? Just for testing.

**Review by VatsalJagani — APPROVED** [review]

**Comment by mahirchavda** [comment]
@VatsalJagani Do we want to change index from windows to _internal for the logs?

File: TA-sysmon-deploy-for-cyences/default/inputs.conf (hunk 2 above)
Comment (mahirchavda) [reply TA-sysmon-deploy-for-cyences/default/inputs.conf:12]: Tested. It is working as expected

**Pull request merged**